Assets/Countly/Plugins/CountlySDK/Services/SessionCountlyService.cs

```diff
@@ -19,11 +19,12 @@ public class SessionCountlyService : AbstractBaseService
         /// </summary>
         /// <returns>bool</returns>
         internal bool IsSessionInitiated { get; private set; }
-
         private readonly LocationService _locationService;
         private readonly EventCountlyService _eventService;
         internal readonly RequestCountlyHelper _requestCountlyHelper;
 
+        bool _isSessionExtendingStopped = false;
```

Comment:
I assume this is working?
lets name this sth more akin to isInternalTimerStopped or sth, as you know it is sth grander than session logic, as naively named it was previously.
also try this without setting it to false. I assume default value is null?